toxicity/test/test_linear_predictor.py

```diff
@@ -0,0 +1,52 @@
+import numbers
+import unittest
+import pathmagic  # noqa
+from linear_predictor import LogisticPredictor
+import utils
+import pandas as pd
+from preprocessing import tf_idf
+
+train_file = "../data/train.csv"
+test_file = "../data/test.csv"
+
+
+class TestLinearPredictor(unittest.TestCase):
+    # We will use a chunk of the dataset of n rows
+    number_of_rows = 1000
+    # Logistic predictor parameters
+    lr_params = {"C": 4, "dual": True}
+
+    def setUp(self, tf_idf_func=True):
+        self.train = pd.read_csv(train_file, nrows=TestLinearPredictor.number_of_rows)
+        self.test = pd.read_csv(test_file, nrows=TestLinearPredictor.number_of_rows)
+        self.y_train = {tag: self.train[tag].values for tag in utils.TAGS}
+        self.logistic_predictor = LogisticPredictor(**TestLinearPredictor.lr_params)
+        if tf_idf_func:
```

Comment:
Does it ever make sense to do anything without `tf_idf`? I propose to remove the argument and always call tf_idf

Reply:
You are right!